Description: Add test-doctests flag
 Add flag to enable/disable the doctest suite. This is needed because doctests
 fail on armel because of #839840.
Author: Ilias Tsitsimpis <iliastsi@debian.org>
---

--- a/servant.cabal
+++ b/servant.cabal
@@ -29,6 +29,10 @@
   type: git
   location: http://github.com/haskell-servant/servant.git
 
+flag test-doctests
+  description: Enable/Disable the doctest suite
+  default: True
+
 custom-setup
   setup-depends:
     base >= 4 && <5,
@@ -172,7 +176,8 @@
 
   type: exitcode-stdio-1.0
   main-is: test/doctests.hs
-  buildable: True
+  if !flag(test-doctests)
+    buildable: False
   default-language: Haskell2010
   ghc-options: -Wall -threaded
   if impl(ghc >= 8.2)
